Gems/Atom/RHI/Code/Include/Atom/RHI/MultiDeviceDrawPacketBuilder.h

```diff
@@ -59,6 +59,8 @@ namespace AZ::RHI
             //! This additional cache is needed since device-specific StreamBufferViews are returned as objects
             //! and the device-specific DrawItem holds a pointer to it.
             AZStd::unordered_map<int, AZStd::vector<StreamBufferView>> m_deviceStreamBufferViews;
+
+            AZStd::unordered_map<int, DrawPacketBuilder::DrawRequest> m_deviceDrawRequests;
```

Comment:
Add a comment about how this structure is used.

Reply:
It was basically a cache that was introduced during a bugfix, but we actually don't need it, so I just removed it.